components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.publisher/src/main/java/org/wso2/carbon/apimgt/usage/publisher/Constants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.wso2.carbon.apimgt.usage.publisher;
+
+/**
+ * analytics related publisher constants
+ */
+public class Constants {
+
+    public static final String RESPONSE_METRIC_NAME = "apim.response";
+    public static final String FAULTY_METRIC_NAME = "apim.faulty";
```

Comment:
shall we make this "apim:fault" to maintain consistency

Reply:
+1

Reply:
Fixed